.github/workflows/integration-test-mgr-suite.yaml

```diff
@@ -1,5 +1,11 @@
 name: Integration test CephMgrSuite
-on: [pull_request]
+on:
+  push:
+    tags:
+      - v*
```

Comment:
Let's just keep the mgr suite in PRs for now, no need to run in master or releases.

Reply:
removed.